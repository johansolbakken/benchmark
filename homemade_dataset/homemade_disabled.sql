USE test_sort_hashjoin_db;

EXPLAIN FORMAT=TREE
SELECT /*+ DISABLE_OPTIMISTIC_HASH_JOIN */
  a.id AS table_a_id,
  a.name,
  a.value,
  b.id AS table_b_id,
  b.description,
  b.timestamp_col
FROM table_a AS a
JOIN table_b AS b
  ON b.a_id = a.id
ORDER BY b.a_id ASC
;
