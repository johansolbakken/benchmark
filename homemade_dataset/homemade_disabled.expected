EXPLAIN
-> Sort: b.a_id  (cost=4.91..4.91 rows=1)
    -> Inner hash join (b.a_id = a.id)  (cost=4.56..4.56 rows=1)
        -> Table scan on a  (cost=0.866..0.866 rows=1)
        -> Hash
            -> Table scan on b  (cost=1.85..1.85 rows=1)

